### PR TITLE
break larger bottom drawer info sections into their own components an…

### PR DESCRIPTION
**Pull request by cbessee**
…d format content. REFS #22

### CODE REVIEW

File: src/components/stage/stage.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+              indicatorColor="primary"
+              textColor="primary">
+          <Tab label="Friday - July 13th" />
+          <Tab label="Saturday - July 14th" />
+          <Tab label="Sunday - July 15th" />
+        </Tabs>
+
+        {value === 0 && <TabContainer>
+          (Beer + Cocktail Garden opens and only Food Trucks available 5 PM-10 PM)<br /><br />
+          <table>
```
Comment (danielduhh) [inline src/components/stage/stage.js]: Getting this javascript error about a missing `<tbody>`

```javascript
Warning: validateDOMNesting(...): <tr> cannot appear as a child of <table>. Add a <tbody> to your code to match the DOM tree generated by the browser.
   in tr (at stage.js:38)
    in table (at stage.js:37)
    in div (created by Typography)
    in Typography (created by WithStyles(Typography))
    in WithStyles(Typography) (at helpers.js:6)
    in TabContainer (at stage.js:35)
    in div (at stage.js:21)
    in Stage (at bottomDrawer.js:141)
    in div (at bottomDrawer.js:96)
    in div (at bottomDrawer.js:85)
    in div (created by Paper)
    in Paper (created by WithStyles(Paper))
    in WithStyles(Paper) (created by Drawer)
    in Transition (created by Slide)
    in EventListener (created by Slide)
    in Slide (created by WithTheme(Slide))
    in WithTheme(Slide) (created by Drawer)
    in RootRef (created by Modal)
    in div (created by Modal)
    in Portal (created by Modal)
    in Modal (created by WithStyles(Modal))
    in WithStyles(Modal) (created by Drawer)
    in Drawer (created by WithStyles(Drawer))
    in WithStyles(Drawer) (created by SwipeableDrawer)
    in SwipeableDrawer (created by WithTheme(SwipeableDrawer))
    in WithTheme(SwipeableDrawer) (at bottomDrawer.js:73)
    in div (at bottomDrawer.js:72)
    in BottomSheet (created by WithStyles(BottomSheet))
    in WithStyles(BottomSheet) (created by Connect(WithStyles(BottomSheet)))
    in Connect(WithStyles(BottomSheet)) (at app.js:45)
    in div (at app.js:42)
    in Provider (at app.js:41)
    in App (at index.js:10)
```

File: src/components/bottomDrawer/bottomDrawer.css [hunk 2]
```diff
@@ -35,6 +35,8 @@
 }
 
 .content-wrapper {
```
Comment (danielduhh) [inline src/components/bottomDrawer/bottomDrawer.css]: `content-wrapper` for non custom info pins has funky styling
screen shot 2018-07-11 at 10 41 38 am

**Comment by danielduhh** [comment]
Also, do you mind moving the bottom drawer specific components inside the `src/bottomDrawer/` folder @cbessee

**Comment by cbessee** [comment]
@danielduhh New commit #e39d06f should resolve all of the above.

**Comment by danielduhh** [comment]
Awesome, looks good to me

**Pull request merged**